bindings/python/Jamfile

```diff
@@ -135,6 +135,7 @@ my-python-extension libtorrent
   : # requirements
     <include>src
     <conditional>@libtorrent_linking
+    <conditional>@asio_src
```

Comment:
I see why you moved it to Jamroot.jam now. But it seems a bit inconvenient to require the user of the library to take this action. I would imagine that if libtorrent would just include asio, the user could still access its symbols, no?
As a last resort, it could at least be declared as usage-requirements, to make the user of libtorrent automatically do this (but I think I'd prefer the former). Perhaps I'm misunderstanding the fundamental problem.